src/sentry/api/endpoints/debug_files.py

```diff
@@ -461,7 +461,7 @@ def delete(self, request, project):
         archive_name = request.GET.get("name")
 
         if archive_name:
-            with transaction.atomic():
+            with transaction.atomic(using=router.db_for_write(ReleaseFile)):
```

Comment:
Is this actually correct? The transaction isn't that much use here anyway, since we're trying to make the creation of the `Release` and `ReleaseFile` atomic, which we can no longer do. One option is to start two transactions, one for each db, and then at least if either throws an exception we'll roll back both. Only case where this fails is if the second transaction fails to commit successfully.

Reply:
Actually this case is fine, we're only writing to one table here